docs/guides/chip_tool_guide.md

```diff
@@ -1391,3 +1391,180 @@ $ ./chip-tool any read-by-id <cluster-ids> <attribute-ids> <destination-id> <end
     ```
     ./chip-tool any read-by-id 0xFFFFFFFF 0xFFFFFFFF 1 0xFFFF
     ```
+
+<hr>
+
+## Saving users and credentials on door lock devices
+
+Matter door lock devices can store pools of users and credentials that allow you to configure different access scenarios. Each user and credential in the pool has an index value. Additionally, each user has a list of Occupied Credentials.
+
+By default, each door lock device comes with no user or credential defined, but it reserves several slots in both pools that can be populated with new users and credentials, up to the value specified in the the `NumberOfTotalUsersSupported` attribute and the `NumberOfCredentialsSupportedPerUser` attribute, respectively.
+
+For security reasons, all communication between users and credentials happens only using their respective index values. No other information is shared between both pools.
+
+The CHIP Tool lets you add users and credentials on door lock devices and securely match their indexes to one another. This is an optional feature available only when working with devices that implement the `doorlock` cluster.
+
+> **Note:**
+> Users and credentials can only be modified by whoever has the right permissions, as specified in the Access Control List.
+
+Complete the following steps:
+
+1. Set up a user on the device.
+1. Assign a credential for the newly created user.
+
+### Step 1: Set up a user
+
+To set up a user on a door lock device with the CHIP Tool, use the following command pattern:
+
+```
+$ ./chip-tool doorlock set-user <OperationType> <UserIndex> <UserName> <UserUniqueID> <UserStatus> <UserType> <CredentialRule> <destination-id> <endpoint-id> --timedInteractionTimeoutMs 1000
+```
+
+In this command:
+
+-   _<OperationType\>_ is one of the available types of operation for the user:
+
+    - `Add` - This operation adds a new user to the slot at _<UserIndex\>_.
+    - `Clear` - This operation removes an existing user from the slot at _<UserIndex\>_.
+    - `Modify` - This operation modifies an existing user at the slot at _<UserIndex\>_.
+
+-   _<UserIndex\>_ is the index value of the user, between `1` and the maximum value you set in the `NumberOfTotalUsersSupported` attribute. Setting the user index to `0` will cause an error.
+-   _<UserName\>_ is the name of the user, which can have maximum 10 bytes of size.
+-   _<UserUniqueID\>_ is a 4-byte number that describes the unique user ID.
+-   _<UserStatus\>_ can be set to one of the following values:
+
+    - Available (`0`) - This status indicates that the given user slot is available for modification on the device.
+    - Occupied (`1`) - This status indicates that the given user slot is used and active.
```

Comment:
I've compared it with the spec and I can see that there are 3 options - 
0 as Available, which is mandatory,
1 as OccupiedEnabled, which is mandatory
3 as OccupiedDisabled, which is optional. 
I'm wondering whether it is worth adding the third option here or not because in real case the user wants to set 0  as available or 1 as occupied.